3.Tests/API/StepDefinitions/CommonSteps.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Text.Json;
+using RestSharp;
+using TechTalk.SpecFlow;
+using Todoly.Core.Helpers;
+using Todoly.Views.Models;
+
+namespace Todoly.Tests.API.Steps.Commons
+{
+    public class CommonSteps
+    {
+        private readonly ScenarioContext _scenarioContext;
+        public readonly RestHelper Client;
+
+        public CommonSteps(ScenarioContext scenarioContext)
+        {
+            Client = new RestHelper(ConfigBuilder.Instance.GetString("api", "ApiHostUrl"));
+            _scenarioContext = scenarioContext;
+        }
+
+        [Given(@"the user has (valid|invalid) credentials")]
+        public void SetCredentials(string credentials)
+        {
+            if (credentials == "valid")
+            {
+                Client.AddAuthenticator(ConfigBuilder.Instance.GetString("TODO-LY-EMAIL"), ConfigBuilder.Instance.GetString("TODO-LY-PASS"));
+            }
+            else
+            {
+                Client.AddAuthenticator("invalidemail@email.com", "");
+            }
+        }
+
+        [Then(@"the API should return a ""(.*)"" response")]
+        public void APIShouldReturnOkResponse(string response)
+        {
+            RestResponse res = (RestResponse)_scenarioContext["Response"];
+            Assert.True(res.IsSuccessful);
+            Assert.Equal(response, res.StatusCode.ToString());
+        }
+
+        [Then(@"a (.*) status code with a ""(.*)"" error message")]
```

Comment:
Keeping the same style of the previews step definition, this one should be something like `Then the API should return a....`